spell-check-word-allow-list.yaml

```diff
@@ -166,3 +166,4 @@ whiteList:
   - utf
   - payu
   - cs
+  - poland
```

Comment:
is this picked up as a typo? weird :-)

Reply:
Country names must start with a capital letter